DiscordChatExporter.Cli/Commands/GetChannelsCommand.cs

```diff
@@ -20,6 +20,12 @@ public class GetChannelsCommand : DiscordCommandBase
     )]
     public required Snowflake GuildId { get; init; }
 
+    [CommandOption(
+        "threads",
+        Description = "Display threads alongside channels."
+    )]
+    public bool ThreadShow { get; init; } = true;
```

Comment:
To keep it consistent with `export all` options, should we name this `--include-threads`/`IncludeThreads`?

https://github.com/Tyrrrz/DiscordChatExporter/blob/9048557b17ab7f51986458c30126260e9bd73761/DiscordChatExporter.Cli/Commands/ExportAllCommand.cs#L19-L29

Reply:
That seems like a good idea. I'll make that change.

Reply:
Resolved in 62a58bb.